packages/fuels-accounts/src/provider/retryable_client.rs

```diff
@@ -171,7 +171,7 @@ impl RetryableClient {
         tx: &[Transaction],
         utxo_validation: Option<bool>,
     ) -> RequestResult<Vec<TransactionExecutionStatus>> {
-        self.wrap(|| self.client.dry_run_opt(tx, utxo_validation))
+        self.wrap(|| self.client.dry_run_opt(tx, utxo_validation, None))
```

Comment:
Maybe @hal3e wants to use gas price=)

Reply:
I think we can skip it for now since i'll be added here #1396